docs/setup.md

```diff
@@ -67,6 +63,10 @@ docker run -it --runtime=habana -v ./llm-on-ray:/root/llm-ray --name="llm-ray-ha
 ```
 
 #### 3. Launch Ray cluster
+If DeepSpeed is enabled, we should dynamic link oneCCL and Intel MPI libraries:
```

Comment:
This is not only needed when Deepspeed is enabled. When you do distributed finetuning, oneCCL is required.